dev/breeze/src/airflow_breeze/utils/parallel.py

```diff
@@ -352,6 +354,11 @@ def check_async_run_results(
     :param include_success_outputs: include outputs of successful parallel runs
     :param poll_time: what's the poll time between checks
     :param skip_cleanup: whether to skip cleanup of temporary files.
+    :param print_summary_after_line_matching: if it is not None, the regexp determines line after which
+        outputs should be printed as summary, so that you do not have to look at the folded details of
+        the run in CI
+    :param print_summary_also_after_success: if summary is printed, this flag determines if summaries should
```

Comment:
```suggestion
    :param print_summary_after_success: if summary is printed, this flag determines if summaries should
```

I think we can drop "also"?

Reply:
Actually It will also print summary after failure. 

The 'print_summary_after_succes" name might be misleading (Yes i had it like that initially and when I used it, i realised that it can be misleading so I added `also`). I could change it to `also_print_summary_after_success` or smth  like that but this is rather small thing :)

Reply:
I'm good either way honestly, that's why I gave it the ✅. Probably just my familiarity bias: "of course it prints it on failure" 🙃.

Reply:
Actually I changed that part to be more explicit:

```
        summarize_on_ci=SummarizeAfter.SUCCESS,
        summary_start_regexp=".*Constraints generated in.*",
```

and 

```
        summarize_on_ci=SummarizeAfter.FAILURE,
        summary_start_regexp=r".*= FAILURES.*|.*= ERRORS.*",
```

Is much more explicit. And this way we have a bit more flexibility (SUCCESS/FAILURE/BOTH).